### ISSUE TITLE
rate_min is limited to less than 4G

### ISSUE THREAD

**Issue by rustyscottweber** [axboe/fio#716]
Fio version is 3.6.  Not terribly out of date, but not up to date.

```
fio jobby.fio 
max value out of range: 4294967296 (4294967295 max)
fio: failed parsing rate_min=4G
fio: job global dropped
```
Job file
```
[global]

# read-only bandwith test
# runs for 100 minutes
# fails when 1GB/s is not met

rw=read
runtime=100m
rate_min=12G

log_avg_msec=1000
write_bw_log=ess-ion
write_iops_log=ess-ion

bs=4m
direct=1
disable_bw_measurement=0
disable_clat=1
disable_lat=1
disable_slat=1
exitall=1
group_reporting
iodepth=32
ioengine=libaio
numjobs=1
overwrite=1
size=100%
thread=0
time_based=1
verify_backlog=0

# run for each FUSIONIO LUN

[jobbyjob-sdb]
filename=/dev/sdb
[jobbyjob-sdc]
filename=/dev/sdc
[jobbyjob-sdd]
filename=/dev/sde
[jobbyjob-sde]
filename=/dev/sde
```
We would like to be able to fail our test should we be reading less than 8 Gibibytes from any disk. 32 Gibibytes for the entire system.

**Comment by sbates130272** [comment]
@sitsofe would you prefer a patch to the mailing list or a PR for this?

**Comment by sitsofe** [comment]
PR will be fine.

**Comment by sbates130272** [comment]
@sitsofe please see the referenced PR. This appears to be working for us on our systems.

**Comment by sitsofe** [comment]
(@sbates130272 Just in case it doesn't get seen: do you think `rate_bytes` will be OK (see https://github.com/axboe/fio/pull/990#issuecomment-631003550 ))

**Comment by axboe** [comment]
@sitsofe On 32-bit it'd need a patch too.

**Comment by sitsofe** [comment]
@axboe ah so that's going to mean 64 bit Windows (LLP64?) needs something too right?

**Comment by axboe** [comment]
Yeah, or weird setups like Windows where unsigned long is 32-bit even on 64-bit environments.

**Comment by sitsofe** [comment]
OK, I guess we'd better re-open this one until a few more patches arrive.

**Comment by axboe** [comment]
@sbates130272 Care to fix that one up, please?

**Comment by sbates130272** [comment]
@axboe @sitsofe Looks like this was addressed in #991. Thanks! LMK if you need anything from me in this regard.

**Comment by axboe** [comment]
Yep, should be all good.
